src/universal/utils/transit.js

```diff
@@ -57,4 +57,32 @@ export default transit.withExtraHandlers([
       : require('url').parse(value)
     ),
   },
-]);
+];
+
+const modifiedHandlers = handlers.withExtraHandlers(extraHandlers);
+
+const reader = transit.reader('json', {
+  handlers: modifiedHandlers.read,
+  cache: false,
+  // Copied from transit-immutable-js. Without this, Error fails to read.
+  mapBuilder: {
+    init() {
+      return {};
+    },
+    add(m, k, v) {
+      // eslint-disable-next-line no-param-reassign
+      m[k] = v;
+      return m;
+    },
+    finalize(m) {
+      return m;
+    },
+  },
+});
+const writer = transit.writer('json', { handlers: modifiedHandlers.write, cache: false });
+
+export default {
+  ...instance.withExtraHandlers(extraHandlers), // added for test cases
```

Comment:
like https://github.com/americanexpress/one-app/blob/main/__tests__/universal/utils/transit.spec.js#L66-L72
huh, are the other methods used? 🤔